pystiche_papers/li_wand_2016/_loss.py

```diff
@@ -48,6 +48,25 @@ def content_loss(
     layer: str = "relu4_2",
     score_weight: Optional[float] = None,
 ) -> FeatureReconstructionOperator:
+    r"""Content_loss from :cite:`LW2016`.
+
+    Args:
+        impl_params: If ``True``, uses the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see below.
+        multi_layer_encoder: Pretrained :class:`~pystiche.enc.MultiLayerEncoder`. If
+            omitted, the default
+            :func:`~pystiche_papers.li_wand_2016.multi_layer_encoder` is used. Defaults
+            to ``None``.
+        layer: Layer from which the encodings of the ``multi_layer_encoder`` should be
+            taken. Defaults to "relu4_2".
+        score_weight: Score weight of the operator. If omitted, the score_weight is
+            determined with respect to ``impl_params``. For details see below.
+
+    If ``impl_params is True`` , a score_weight of 2e1 is used instead of 1e0, and a
```

Comment:
Couldn't we move the first part into the description of `score_weight`? Maybe we should reserve the part below the `Args` for stuff that can't be explained within.

Reply:
Isn't it better to describe the influence of the parameter `impl_params` on the parameters uniformly in the block below `Args`?

Reply:
I think `Args` is the right place to describe everything related to the input arguments. For example, it bothers me that I don't know the default value for `score_weight` without reading on after the `Args` section.